odoo/addons/test_http/tests/test_session.py

```diff
@@ -213,3 +217,40 @@ def test_session7_serializable(self):
             with self.assertRaises(TypeError):
                 dict.update(session, foo=value)
             self.assertFalse(session.foo)
+
+
+class TestSessionStore(HttpCaseWithUserDemo):
+    def setUp(self):
+        super().setUp()
+        tmpdir = TemporaryDirectory()
```

Comment:
better not create a new tempdir for every single test (in case we add more tests in the future) => should be inside a setUpClass